webui/react/src/hooks/useTheme.ts

```diff
@@ -7,6 +7,7 @@ import { BrandingType } from 'types';
 
 type ThemeHook = {
   mode: DarkLight,
+  scheme: DarkLight,
```

Comment:
**chore:** if we update `mode` to handle `Auto` (see comment below), we should be ok without `scheme` as a prop.